.changeset/ten-baboons-compete.md

```diff
@@ -0,0 +1,7 @@
+---
+"@eth-optimism/contracts": minor
```

Comment:
Some of these changes are presumably API breaking, but I'm not sure what the current norms are and if we want to be make major bumps all that often right now.

Reply:
Contracts are pre-1.0 so minor is OK